### DISCUSSION TITLE
[自动摘要] 自动截断前面几百字生成文章摘要

### DISCUSSION THREAD

**Discussion by JIeJaitt** [tangly1024/NotionNext#2062] [Ideas]
**为什么提出这个新的特性改动**
每篇文章都自己手动填写摘要显得太繁琐，希望能够类似 `Hexo` 那种形式直接截断文章的前面几百字生成摘要

**描述一下你推荐的解决方案**
摘取文章的前面几百字自动生成摘要

**描述一下你考虑过的其它替代解决方案**
无

**补充说明**
无

**Comment by tangly1024** [comment]
目前推荐手动完善摘要或者借助ChatGPT编写内容摘要，自动截取的效果可能并不好

**Comment by JIeJaitt** [comment]
> 目前推荐手动完善摘要或者借助ChatGPT编写内容摘要，自动截取的效果可能并不好

主要是考虑方便迁移而且也不麻烦，当hexo的文章迁移过来的时候，文章中的`<!-- more -->`能够自动截断`<!-- more -->`前部分的字作为摘要。

如果使用`ChatGPT`我认为有几个缺点，一个是麻烦，调用api之类啥的；二个是价钱不便宜，之前偶然看过heo大佬的博客用过这个，瞅了一眼没记住价格，但是这显然违背了notion next的设计初衷。另外我感觉还会拖慢文章的加载速度。
